backend/users/views/user.py

```diff
@@ -88,6 +93,12 @@ def has_permission(self, request, view):
         return request.user.channel.role == UserChannel.Role.ADMIN
 
 
+class IsAdmin(permissions.BasePermission):
+    def has_permission(self, request, view):
+        if request.user.is_authenticated:
+            return request.user.channel.role == UserChannel.Role.ADMIN
```

Comment:
Do we need to check `if request.user.is_authenticated:` ? It looks like `IsAdminOrReadOnly` doesn't